workflows/dnaseq/dnaseq-standard-fastq.wdl

```diff
@@ -0,0 +1,114 @@
+## **WARNING:** this workflow is experimental! Use at your own risk!
+
+version 1.1
+
+import "../../data_structures/read_group.wdl"
+import "../../tools/fq.wdl"
+import "./dnaseq-core.wdl" as dnaseq_core_wf
+
+workflow dnaseq_standard_fastq_experimental {
+    meta {
+        description: "Aligns DNA reads using bwa"
+        outputs: {
+            harmonized_bam: "Harmonized DNA-Seq BAM, aligned with bwa",
+            harmonized_bam_index: "Index for the harmonized DNA-Seq BAM file"
+        }
+        allowNestedInputs: true
+    }
+    parameter_meta {
+        read_one_fastqs_gz: "Input gzipped FASTQ format file(s) with 1st read in pair to align"
+        read_two_fastqs_gz: "Input gzipped FASTQ format file(s) with 2nd read in pair to align"
+        bwa_db: "Gzipped tar archive of the bwa reference files. Files should be at the root of the archive."
+        reads_per_file: "Number of reads per FASTQ file to output."
+        read_groups: {
+            description: "An Array of structs defining read groups to include in the harmonized BAM. Must correspond to input FASTQs. Each read group ID must be contained in the basename of a FASTQ file or pair of FASTQ files if Paired-End. This requirement means the length of `read_groups` must equal the length of `read_one_fastqs_gz` and the length of `read_two_fastqs_gz` if non-zero. Only the `ID` field is required, and it must be unique for each read group defined. See top of file for help formatting your input JSON.",  # TODO handle unknown RG case
+            external_help: "https://samtools.github.io/hts-specs/SAMv1.pdf",
+            fields: {
```

Comment:
IMO this should not be repeated. We've written these definitions in `read_group.wdl` and we (or maybe just I) don't want it in multiple places.